explanation/dnssec/dnssec.md

```diff
@@ -0,0 +1,254 @@
+(dnssec)=
+# DNS Security Extensions (DNSSEC)
+
+DNS is a mapping system between names and IP addresses. Besides this main purpose that allows us to use friendly names instead of a sequence of numbers to reach a web site, it may store more information about a particular domain, such as:
+ * point of contact
+ * when it was last updated
+ * what are the authoritative name servers for the domain
+ * what are the mail exchangers for the domain (i.e., which systems are responsible for email for this domain)
+ * mapping of host names to IP addresses, and vice-versa
+ * information about other services, such as kerberos realms, LDAP servers, etc (usually on internal domains only)
+ * and more
+
+When DNS was first conceived, security wasn't a top priority yet. At its origins, DNS is susceptible to multiple vulnerabilities, and has many weaknesses. Most of them are a consequence of spoofing: there is no guarantee that the reply you got to a DNS query a) was not tampered with; b) came from the true source.
```

Comment:
'yet' can be removed from the first sentence.

s/reply you got/reply you received/

Reply:
good